client/state/plugins/wporg/actions.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+const debug = require( 'debug' )( 'calypso:wporg-data:actions' );
+
+/**
+ * Internal dependencies
+ */
+import wporg from 'lib/wporg';
+import utils from 'lib/plugins/utils';
+import { RECEIVE_WPORG_PLUGIN_DATA, FETCH_WPORG_PLUGIN_DATA } from 'state/action-types';
+
+/**
+ *  Local variables;
+ */
+let _fetching = {};
+
+export default {
+	fetchPluginData: function( pluginSlug ) {
+		return ( dispatch ) => {
+			if ( _fetching[ pluginSlug ] ) {
+				return;
+			}
+			_fetching[ pluginSlug ] = true;
+
+			setTimeout( () => {
```

Comment:
Why do we need to encapsulate this with `setTimeout`? 
Looking at something like https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/plans/actions.js#L42 which doesn't seem to do that.